.github/workflows/main.yml

```diff
@@ -0,0 +1,49 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build APK
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Update MasterList file
```

Comment:
Is it possible to make this as a single stream command? Something like this:
```
wget "..." | unzip | openssl ... -out app/src/main/assets/masterList.asn
```

Reply:
I've updated the file with the new command